java/shared-utils/src/main/java/com/newrelic/shared/OpenTelemetryConfig.java

```diff
@@ -70,7 +74,16 @@ public static OpenTelemetrySdk configureGlobal(String defaultServiceName) {
     }
 
     // Configure metrics
-    var meterProviderBuilder = SdkMeterProvider.builder().setResource(resource);
+    var meterProviderBuilder =
+        SdkMeterProvider.builder()
+            .setResource(resource)
+            // Aggregate OBSERVABLE_UP_DOWN_SUM as gauge instead of sum. This allows OBSERVABLE_UP_DOWN_SUM
+            // data to still be useful when aggregation temporality is set to DELTA.
+            .registerView(
+                InstrumentSelector.builder()
+                    .setInstrumentType(InstrumentType.OBSERVABLE_UP_DOWN_SUM)
```

Comment:
We probably want to demonstrate changing the aggregation to last value for `InstrumentType.UP_DOWN_COUNTER` since those will be equally unusable as deltas.

... Also is it `OBSERVABLE_UP_DOWN_SUM` or `OBSERVABLE_UP_DOWN_COUNTER`?

Reply:
Last value doesn't help for sync up down counter because we would just get the last incremental recording. I.e. +1 or -1 or something liek that. 

>... Also is it OBSERVABLE_UP_DOWN_SUM or OBSERVABLE_UP_DOWN_COUNTER?

Its both. We recently deprecated `OBSERVABLE_UP_DOWN_SUM` in favor of `OBSERVABLE_UP_DOWN_COUNTER`

Reply:
> Last value doesn't help for sync up down counter because we would just get the last incremental recording. I.e. +1 or -1 or something liek that.

I promise, you'll only have to say this 16 more times.